spec/gitlab/client/notes_spec.rb

```diff
@@ -152,5 +152,22 @@
         expect(@note.author.name).to eq("John Smith")
       end
     end
+
+    context "when merge_request note" do
+      before do
+        stub_post("/projects/3/merge_request/7/notes", "note")
```

Comment:
I think the test failed because you stubbed `merge_request` in the path but it really should be `merge_requests`.